content/docs/user-guide/external-dependencies.md

```diff
@@ -146,15 +146,18 @@ $ dvc run -n download_file \
 
 </details>
 
-## Example: DVC remote aliases
+## Example: Using DVC remote aliases
 
-If instead of a URL you'd like to use an alias that can be managed
-independently, or if the external dependency location requires access
-credentials, you may use `dvc remote add` to define this location as a DVC
-Remote, and then use a special URL with format `remote://{remote_name}/{path}`
-to define an external dependency.
+You may want to encapsulate external locations as configurable entities that can
+be managed independently. This is useful if multiple dependencies (or stages)
+reuse the same location, or if its likely to change in the future. And if the
+location requires authentication, you need a way to configuring it in order to
+access the data.
```

Comment:
also users might redefine using `--local` config, thus using their own external space

Reply:
True. That seems a too `dvc remote`-specific to add in this paragraph though, IMO. The concept of "DVC remote" isn't even mentioned yet, and it's not really explained in this section.

Reply:
BUT I could change the example from HTTP to something that requires setting up credentials to give a hint of that possibility... ⌛

Reply:
Done in d16cd33d.